documentation/modules/con-overview-open-api-spec-kafka-bridge.adoc

```diff
@@ -13,7 +13,7 @@ The Kafka Bridge OpenAPI specification is in JSON format.
 You can find the OpenAPI JSON files in the `src/main/resources/` folder of the Kafka Bridge source download files.
 The download files are available from the {ReleaseDownload}.
 
-You can also use the xref:_openapi[`GET /openapi` method] to retrieve the OpenAPI v2 specification in JSON format.
+You can also use the xref:openapi[`GET /openapi` method] to retrieve the OpenAPI v2 specification in JSON format.
```

Comment:
Do we want to talk about the V2 and v3 endpoints here and guide users to v3?

Reply:
I guess these xrefs work okay from IDs in HTML output?  In the index.adoc, the refs have a period `[.openapi]`. Same with the `commit` xrefs etc

Reply:
> Do we want to talk about the V2 and v3 endpoints here and guide users to v3?

Take a note please and let's leave this change for a different PR. For this one, let's focus on having bridge doc back again.

> I guess these xrefs work okay from IDs in HTML output? In the index.adoc, the refs have a period [.openapi]. Same with the commit xrefs etc

The new generator generates the section headers with the period instead of the underscore but I also noticed that if I put the period here ... the link doesn't work. I had to remove it. Is it some asciidoc stuff you can confirm?

Reply:
As agreed offline with Paul everything seems to be fine.